IPython/core/interactiveshell.py

```diff
@@ -2664,7 +2664,7 @@ def safe_execfile(self, fname, *where, **kw):
                 # tb offset is 2 because we wrap execfile
                 self.showtraceback(tb_offset=2)
 
-    def safe_execfile_ipy(self, fname, shell_futures=False):
+    def safe_execfile_ipy(self, fname, raise_exceptions=False, shell_futures=False):
```

Comment:
Inserting this here will break user code that was previously calling `safe_execfile_ipy(name, True)`, not explicitly setting the kwarg.